frontend/templates/views/partials/header.j2

```diff
@@ -19,24 +19,137 @@
     </a>
 
     {% do doc.styles.addCssFile('/css/components/molecules/nav-link.css') %}
-    <nav class="ap-o-header-first-level">
+    <nav class="ap-o-header-main">
       {# Sections #}
+
       {% for section in g.collection('amp-dev').collections()|sort(attribute='order') %}
-      {% if section.index %}
 
-      {% set section_index = g.doc(section.index, locale=doc.locale) %}
-      <a class="ap-o-header-first-level-link ap-m-nav-link {{ active_class(section_index) }}" href="{{ section_index.url.path }}">
-        {{ section.title }}
-      </a>
-      {% elif section.external_index %}
-      <a class="ap-o-header-first-level-link ap-m-nav-link" href="{{ section.external_index }}">
-        {{ section.title }}
-      </a>
-      {% endif %}
-      {% endfor %}
-    </nav>
+      {# hover is not avaiable for larger touch devices, a work around for displaying the nav list is using hidden input fields #}
+      <input id="main-item-radio-{{ section.order }}" type="radio" name="ap-o-header-main-item-radio-btn" role="button" tabindex="0" />
+      
+      <div class="ap-o-header-main-item">
+        {% if section.index %}
+        {% set section_index = g.doc(section.index, locale=doc.locale) %}
+          <a class="ap-o-header-main-link ap-m-nav-link {{ active_class(section_index) }}" href="{{ section_index.url.path }}">
+            {{ section.title }}
+          </a>          
+
+          {% if section.sub_navigation %}
+            {% do doc.icons.useIcon('/icons/angle-down-solid.svg') %}
+            <label class="ap-a-ico ap-o-header-main-link-icon" for="main-item-radio-{{ section.order }}">                
+                <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#angle-down-solid"></use></svg>
+            </label>
+          {% endif %}
+
+        {% elif section.external_index %}
+          <a class="ap-o-header-main-link ap-m-nav-link" href="{{ section.external_index }}">
+            {{ section.title }}
+          </a>  
+          {% if section.sub_navigation %}
+            {% do doc.icons.useIcon('/icons/angle-down-solid.svg') %}
+              <label class="ap-a-ico ap-o-header-main-link-icon" for="main-item-radio-{{ section.order }}">
+                <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#angle-down-solid"></use></svg>
+              </label>
+          {% endif %}
+        {% endif %}
+
+        {% if section.sub_navigation %}
+        <ul class="ap-o-header-flyout">
+          <div class="ap-o-header-flyout-fake-shadow"></div>
+          {% for second_level_path in section.sub_navigation %}
+            {% set second_level_doc = g.doc(second_level_path, locale=doc.locale) %}
+            {% if second_level_doc.collection.collections() and second_level_doc.collection.title != "Guides & Tutorials" %}
```

Comment:
Please don't check for a `title` that's not reliable. Can be changed and/or localized.